src/radical/pilot/raptor/master.py

```diff
@@ -163,11 +166,9 @@ def _get_config(self, cfg=None):
         cfg['log_lvl'] = 'warn'
         cfg['kind']    = 'master'
         cfg['base']    = pwd
-        cfg['sid']     = os.environ['RP_SESSION_ID']
-        cfg['base']    = os.environ['RP_PILOT_SANDBOX']
-
-        # FIXME: use registry for comm EP info exchange, not cfg files
-      # cfg['path'] = os.environ['RP_TASK_SANDBOX']
+        cfg['sid']     = self._sid
+        cfg['base']    = self._sbox
```

Comment:
then line above `cfg['base']    = pwd` should be removed?
and line above that should be updated
```python
ru.dict_merge(cfg, ru.read_json('%s/control_pubsub.json' % self._psbox))
```
and `pwd` is not needed

Reply:
Ah, of course, I removed the duplication.  Ack on the merge path, fixed.